Base.rte/Scenes/Terrains/Terrains.ini

```diff
@@ -149,6 +149,14 @@ Terrain = SLTerrain
 		Y = -1
 	BackgroundTexture = ContentFile
 		FilePath = Base.rte/Scenes/Textures/DirtDark.png
+	AddTerrainFrosting = TerrainFrosting
+		TargetMaterial = Material
+			CopyOf = Sand
+		FrostingMaterial = Material
+			CopyOf = Sand
```

Comment:
Frosting sand with sand? Ok I guess.

Reply:
It prevents every step instantly creating ugly holes where the BG layer is visible. I did the same thing with snow.

Think of it as sand/snow that has been moved around by the wind.